spec/oas3.patched/components/schemas/Device.yaml

```diff
@@ -108,31 +108,7 @@ properties:
       $ref: './Href.yaml'
     type: array
   state:
-    type: string
-    enum:
-      - queued
-      - provisioning
-      - deprovisioning
-      - reinstalling
-      - active
-      - inactive
-      - failed
-      - powering_on
-      - powering_off
-      - deleted
-    description: |-
-      The current state the instance is in.
-
-      * When an instance is initially created it will be in the `queued` state until it is picked up by the provisioner.
-      * Once provisioning has begun on the instance it's state will move to `provisioning`.
-      * When an instance is deleted, it will move to `deprovisioning` state until the deprovision is completed and the instance state moves to `deleted`.
-      * If an instance fails to provision or deprovision it will move to `failed` state.
-      * Once an instance has completed provisioning it will move to `active` state.
-      * If an instance is currently powering off or powering on it will move to `powering_off` or `powering_on` states respectively. 
-      * When the instance is powered off completely it will move to the `inactive` state.
-      * When an instance is powered on completely it will move to the `active` state.
-      * Using the reinstall action to install a new OS on the instance will cause the instance state to change to `reinstalling`.
-      * When the reinstall action is complete the instance will move to `active` state.
+    $ref: './DeviceStateFlexible.yaml'
```

Comment:
We could write a small script to automate this (and upstream those changes and that helper).